src/ecs/rendering/bundle.rs

```diff
@@ -92,10 +95,14 @@ where
             up: [0.0, 1.0, 0.0].into(),
         };
 
+        let window_events = EventHandler::<WindowModifierEvent>::new();
```

Comment:
`new` defaults to a buffer size of 200, I never see us needing more than 10 for this so use `with_capacity` instead.

Reply:
Make it 20 just top be on the safe side